src/geant4/tests/dagsolid_unit_tests.cpp

```diff
@@ -118,6 +118,83 @@ TEST_F(DagSolidTest, point_outside_volume) {
   return;
 }
 
+/*
+ * surface normal test - test all 6 faces of the cube
+ */
+TEST_F(DagSolidTest, surface_normal_test) {
+  // set the vector to be the rhs
+  G4ThreeVector position = G4ThreeVector(50., 0., 0.);
+
+  G4ThreeVector normal = vol_1->SurfaceNormal(position);
+
+  EXPECT_EQ(normal[0], 1.0);
+  EXPECT_EQ(normal[1], 0.0);
+  EXPECT_EQ(normal[2], 0.0);
+
+  // set the vector to be the lhs
+  position.setX(-50.);
+
+  normal = vol_1->SurfaceNormal(position);
+
+  EXPECT_EQ(normal[0], -1.0);
+  EXPECT_EQ(normal[1], 0.0);
+  EXPECT_EQ(normal[2], 0.0);
+
+  // set the vector to be the lhs
```

Comment:
Update comment?
```suggestion
  // set the vector to be in front
```
(Or maybe on top??)

Reply:
Done